RoddyCore/src/de/dkfz/roddy/config/RoddyAppConfig.java

```diff
@@ -0,0 +1,90 @@
+package de.dkfz.roddy.config;
+
+import de.dkfz.roddy.RunMode;
+import de.dkfz.roddy.tools.AppConfig;
+import de.dkfz.roddy.tools.RoddyConversionHelperMethods;
+
+import java.io.File;
+
+public class RoddyAppConfig extends AppConfig {
+
+    public RoddyAppConfig() {
+        super();
+    }
+
+    public RoddyAppConfig(File file) {
+        super(file);
+    }
+
+    public String getOrSetApplicationProperty(RunMode runMode, String pName) {
+        return getOrSetApplicationProperty(runMode, pName, "");
+    }
+
+    public String getOrSetApplicationProperty(RunMode runMode, String pName, String defaultValue) {
+        return getOrSetApplicationProperty(runMode.name() + "." + pName,
+                getOrSetApplicationProperty(pName, defaultValue));
+    }
+
+    public boolean getOrSetBooleanApplicationProperty(String pName, boolean defaultValue) throws ConfigurationError {
+        try {
+            return Boolean.parseBoolean(getOrSetApplicationProperty(pName, String.valueOf(defaultValue), "boolean"));
```

Comment:
Maybe introduce an enum for the type?

Reply:
Yes, I was also thinking about that. But then I thought we should first get a better overview about where such code is located, and then centralize this kind of code with a good conceptual idea in the background. So I left further refactoring for later.

Reply:
Sure